content/docs/2.12/scalers/prometheus.md

```diff
@@ -32,6 +33,7 @@ triggers:
 **Parameter list:**
 
 - `serverAddress` - Address of Prometheus server. If using VictoriaMetrics cluster version, set full URL to Prometheus querying API, e.g. `http://<vmselect>:8481/select/0/prometheus`
+- `serverAPIPath` - API path to query the Prometheus Server. On Grafana Cloud, this would be `/api/prom`. (Default: `/api/v1/query`)
```

Comment:
```suggestion
- `serverAPIPath` - API path to query the Prometheus Server. (Default: "/api/v1/query", Optional, ex: On Grafana Cloud, this would be `/api/prom`)
```

Reply:
I have updated a bit the style to follow the style guide 😄